src/app/service/guest-pass/GuestPassService.ts

```diff
@@ -66,7 +66,7 @@ export default async (client: DiscordClient): Promise<void> => {
 	}
 
 	// Begin reminder of active guest users
-	for (const activeUser of listOfActiveGuests) {
+	listOfActiveGuests.forEach(activeUser => {
```

Comment:
Any reason why this was changed?

Reply:
not me, must be from recent merge of devlop

Reply:
oh ok in that case can you please change it back so that it matches the develop branch?